modules/statics/src/map.ts

```diff
@@ -59,4 +59,8 @@ export class CoinMap {
   public forEach(callback: (coin: Readonly<BaseCoin>, coinName: string) => void): void {
     this._map.forEach(callback);
   }
+
+  public [Symbol.iterator](): IterableIterator<[string, Readonly<BaseCoin>]> {
```

Comment:
What is this needed/used for?

Reply:
https://github.com/BitGo/BitGoJS/pull/1923/commits/a4a2f4b830084a136840abbaf3b30fe5852a60e1